setup.py

```diff
@@ -398,6 +398,7 @@ def do_setup():
             'Programming Language :: Python :: 2.7',
             'Programming Language :: Python :: 3.4',
```

Comment:
It looks like this line should be dropped since 3.4 is not tested in the CI process

Reply:
disregard - this has been fixed in https://github.com/apache/incubator-airflow/pull/4259